lib/runnerman.js

```diff
@@ -1,18 +1,19 @@
 const { Collection, Event, Script } = require('postman-collection');
 const { run: newman } = require('newman');
+const { format } = require('util');
+const { v4: uuidv4 } = require('uuid');
 const { filterCollection } = require('./util/filterCollection');
 const { generateGlobals } = require('./util/generateGlobal');
-const { format } = require('util');
 const { createGlobalVariable } = require('./util/createGlobalVariable');
-const { v4: uuidv4 } = require('uuid');
 
-module.exports = async function (suite, collectionDefinition, environment, iterations) {
+
+module.exports = async function runnerman(suite, collectionDefinition, environment, iterationCount, reporters) {
```

Comment:
Como feito em https://github.com/FCesar/runnerman/pull/1, crie as funções antes e exponha um objeto contendo as funções (legilibildade).

Reply:
So, I like use this mode.

https://github.com/FCesar/runnerman/blob/a0f3adb4320886d0ad945293175bb045bc478bb0/lib/runnerman.js#L10-L12